piqe_ocp_lib/tests/resources/test_ocp_health_checker.py

```diff
@@ -23,15 +23,15 @@ def test_check_node_health(self, ocp_health):
         all_nodes_healthy, node_health_info = ocp_health.check_node_health()
         assert isinstance(all_nodes_healthy, bool)
         assert isinstance(node_health_info, dict)
-        # if all_nodes_healthy is False, There has to be values of failure nodes in node_health_info dict
+        # if all_nodes_healthy is False, There has to be >1 values of failure nodes in node_health_info dict
         if all_nodes_healthy:
             for key, value in node_health_info.items():
-                assert len(value) == 0
+                assert len(value) == 1
         else:
             failure_values = list()
             for key, value in node_health_info.items():
                 failure_values.append(value)
-            assert len(value) > 0
+            assert len(value) > 1
```

Comment:
We should assert on failure_values list. This list holds if there are any failures.

Reply:
Yes, will fix.

Reply:
Fixed